src/LivewireServiceProvider.php

```diff
@@ -243,7 +243,7 @@ protected function registerViewMacros()
         // Early versions of Laravel 7.x don't have this method.
         if (method_exists(ComponentAttributeBag::class, 'macro')) {
             ComponentAttributeBag::macro('wire', function ($name) {
-                $entries = head($this->whereStartsWith('wire:'.$name));
+                $entries = $this->whereStartsWith('wire:'.$name)->first();
```

Comment:
Actually it looks like `head()` did not have the effect to take the first but just convert to array.
```suggestion
                $entries = iterator_to_array($this->whereStartsWith('wire:'.$name));
```